app/serializers/auction_with_bid_serializer.rb

```diff
@@ -0,0 +1,11 @@
+class AuctionWithBidSerializer
+  def self.serialize(auction_with_bid)
+    auction = auction_with_bid.auction.attributes
+
+    if auction_with_bid.bid
+      auction.merge(bid: auction_with_bid.bid.attributes)
```

Comment:
Once we get the bids, make sure we don't query it twice by calling the method here and above